### DISCUSSION TITLE
How can I force the agent to read the 200 (http) line from a log file?

### DISCUSSION THREAD

**Discussion by Walsounet** [wazuh/wazuh#15910] [Q&A] [answered]
Hello!
Being new to Wazuh, I have some questions. I looked, but I couldn't find an answer to my problem.

I'm using one agent to read the logs from the nginx server that runs my app. I'd like to view the 200 ok log lines on the wazuh dashboard events, but only the 500 (http) lines appear.

**Example (log line)**
127.0.0.1 - - [15/Jan/2023:23:51:16 +0000] "GET /test/login HTTP/1.1" 200 615 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36"

As soon as I switch from a 200 to a 500, the Azure agent notices and logs into the events.
Thank you

**Comment by carlosdams** [comment, marked as answer]
Hi @Walsounet !
I hope you are enjoying Wazuh and welcome to the open source community of it!

I was checking your question and the reason you do not get an alert for the log event (thanks for providing it): 
```
127.0.0.1 - - [15/Jan/2023:23:51:16 +0000] "GET /test/login HTTP/1.1" 200 615 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36"
```
it is because is considered with no security relevance as in [Rules classification - Wazuh](https://documentation.wazuh.com/current/user-manual/ruleset/rules-classification.html)

You can overwrite the rule to increase its level, I would not recommend you to do this because you could get a bunch of events in Wazuh Dashboard that might just be noise, but still if you want to do it I will explain the process:

**1-**  Check if there is any rule that is triggered by the event using the Ruleset Test under the Wazuh Menu > Tools, as you can see it triggers `rule id 31108`
![image](https://user-images.githubusercontent.com/35348824/212685908-7a8d0b5f-0197-4a00-be31-15eb816d3cca.png)

**2-** Search for this rule under Management > Rules and click on the name of the file
![image](https://user-images.githubusercontent.com/35348824/212686252-43796ab0-c2e1-4edd-94e4-a12d75272db6.png)

**3-** Copy the rule from the file, in this case it is file `0245-web_rules.xml`

**4-** From the same page Management > Rules, click on Custom rules and edit the `local_rules.xml` file

**5-** Add this rule with a level 3 or higher (level 3 is the default rule level to generate alerts), add an `overwrite="yes"`, save the file and restart the Wazuh server

After saving the file you can test from the Ruleset Test if the event will trigger an alert
![image](https://user-images.githubusercontent.com/35348824/212687184-87360dc6-7b52-437f-9a3f-56d9a1f39979.png)

More information in the following link: [Custom rules and decoders](https://documentation.wazuh.com/current/user-manual/ruleset/custom.html)

I hope this addresses your question, please let me know

**Reply by Walsounet** [reply]
Thank you, that is precisely what I needed! I might not be approaching my issue the appropriate way, as you suggest, but your method immediately resolved my issue.

Thank you!
